### PR TITLE
Dockerrize + Get settings from vault

### PR DESCRIPTION
**Pull request by rhirsh-twistbio**
@noamby

### CODE REVIEW

File: .drone.yml [hunk 1]
```diff
@@ -0,0 +1,21 @@
+pipeline:
+  docker:
```
Comment (ZivRechnitser) [inline .drone.yml:2]: @rhirsh-twistbio please change the docker plugin to this - https://github.com/Twistbioscience/Twist-API/blob/develop/.drone.yml#L21

File: docker-compose.yml [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+    environment:
+      - VAULT_USER
+      - VAULT_PASSWORD
+    command: celery -A sfconreset worker -l info -B -c20
+    depends_on:
+      - redis
+    networks:
+      - tapi_network
+
+  redis:
```
Comment (ZivRechnitser) [inline docker-compose.yml:17]: @rhirsh-twistbio how should your app know about Redis?

Comment (rhirsh-twistbio) [reply docker-compose.yml:17]: @zivbaz it either resolves to whatever is in OS ENV variables: BROKER_URL and RESULT_BACKEND

OR 

by default it looks for redis at hostname *redis*

File: .drone.yml (hunk 1 above)
Comment (rhirsh-twistbio) [reply .drone.yml:2]: @zivbaz done

**Pull request closed without merging**